Neos.Neos/Configuration/Testing/NodeTypes.yaml

```diff
@@ -7,7 +7,7 @@
 
 'Acme.Demo:ThreeColumn':
   superTypes:
-    'Neos.NodeTypes:Column': true
+    'Neos.Neos:Content': true
```

Comment:
Why this change?

Reply:
Because there is no Neos.Nodetypes:Column any more. Also the tests should not rely on NeosNodetypes at all as we would really like to get rid of this package.

Reply:
@bwaidelich is this ok for you, i would like to merge this as there are 3 approvals already.

Reply:
Yes sure! And thanks for the explanation!